### DISCUSSION TITLE
looking for pdf annotation app

### DISCUSSION THREAD

**Discussion by Burnaviour** [OCA/odoo-community.org#165] [Q&A]
hi there i'm looking for pdf annotation app  is there any app already exist that add annotation to pdf in odoo it would be helpful

**Comment by rousseldenis** [comment]
Hi,

Maybe look at this if it fits your need:
https://github.com/OCA/reporting-engine/tree/14.0/report_fillpdf

Either, you can look in that repository for further modules.

On Thu, Apr 25, 2024 at 6:40 AM Burnaviour ***@***.***> wrote:

> hi there i'm looking for pdf annotation app is there any app already exist
> that add annotation to pdf in odoo it would be helpful
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/OCA/discussions/165>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEU76PKMDTQFBKJTSFCWX7LY7CCLLAVCNFSM6AAAAABGYDN5TSVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGU2TOMZRG4>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 


*Denis Roussel**Software Engineer*
*T    : **+32 2 888 31 49 <%2B32%202%20888%2031%2049>*
*M : **+32 472 22 00 57 <%2B32%20472%2022%2000%2057>*

* <%2B32%20472%2022%2000%2057>*
<https://www.acsone.eu/>
Val Benoit, Quai Banning 6 | B-4000 Liège | Belgium
Atrium Building, Drève Richelle 167 | B-1410 Waterloo | Belgium
Zone industrielle 22 | L-8287 Kehlen | Luxembourg

**Reply by Burnaviour** [reply]
is it available in odoo17 Ver?
